contrib/codeberg-pages/template.py

```diff
@@ -0,0 +1,57 @@
+pkgname = "codeberg-pages"
+pkgver = "5.1"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = [
+    "go",
+]
+makedepends = [
+    "sqlite-devel",
+]
+pkgdesc = "Serve static pages from Gitea/Forgejo repositories"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "EUPL-1.2"
+url = "https://docs.codeberg.org/codeberg-pages"
+source = [
+    f"https://codeberg.org/Codeberg/pages-server/archive/v{pkgver}.tar.gz",
+    "https://github.com/mattn/go-sqlite3/archive/refs/tags/v1.14.22.tar.gz",
+]
+source_paths = [".", "go-sqlite3-patched"]
+sha256 = [
+    "7b26ab1b2fd8e2e8f303351309a83133614d0f31b3cfd8a4edd98469da37bd73",
+    "5b1d74ec4359b1ae0fe373fab37ae8a661ed128cf2f89b27875ecbb18bbe1078",
+]
+
+
+def post_extract(self):
+    # Test depends on a network connection
+    self.rm(self.srcdir / "server/handler/handler_test.go")
+
+
+def do_prepare(self):
+    from cbuild.util import golang
+
+    golang.Golang(self).mod_download()
+
+    # Replace the go dep
+    self.do(
+        "go",
+        "mod",
+        "edit",
+        "-replace",
+        f"github.com/mattn/go-sqlite3@v1.14.16={self.chroot_srcdir / 'go-sqlite3-patched'}",
+    )
+
+
+def init_build(self):
+    from cbuild.util import golang
+
+    self.make_env.update(golang.get_go_env(self))
+    self.make_env["EXTRA_GOFLAGS"] = f"{self.get_goflags(shell=True)} -trimpath"
+    # https://github.com/golang/go/issues/64875
+    self.make_env["CGO_ENABLED"] = "1"
+    self.make_env["TAGS"] = "libsqlite3 sqlite sqlite_unlock_notify"
```

Comment:
this stuff is specific to the forgejo template (as it uses a make file for building instead of the go build style), what you need instead is setting
`go_build_tags = ["libsqlite3", "sqlite", "sqlite_unlock_notify"]` and also `env = {"CGO_ENABLED": "1"}`